src/cogent3/app/io_new.py

```diff
@@ -432,9 +432,12 @@ def main(self, /, data: SerialisableType, *, identifier=None) -> T:
         identifier
         """
         identifier = identifier or get_data_source(data)
```

Comment:
Here we should use `get_data_source`  or new `open_data_source` ?

Reply:
Will move to `open_data_source` when that is merged in

Reply:
Actually, read that closer now. It's not `get_data_source` not **store**.

Reply:
I get confused by those too! But we have `open_data_store` I won't get confused.